jenkins-pipelines/openshift/Jenkinsfile

```diff
@@ -1,7 +1,7 @@
 node ('openstack-jenkins-slave') {
 
     def openshiftHeatBranch = 'master'
-    def openshiftDeploymentAnsibleBranch = 'master'
+    def openshiftDeploymentAnsibleBranch = 'feature/remove-monitoring-test'
```

Comment:
Surely we don't want to merge this branch change?

Reply:
I have now reverted the branch name, ready for merge.